addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -247,7 +247,12 @@ function factory(dependencies) {
          * @private
          */
         _computeTopBar() {
-            return this.hasTopbar ? insertAndReplace() : clear();
+            if (!this.hasTopbar) {
+                return clear();
+            }
+            return insertAndReplace({
+                pendingGuestName: this.messaging.currentGuest && this.messaging.currentGuest.name,
```

Comment:
This does not look correct as a compute, `pendingGuestName` should be written once, on starting to edit.